direct/src/dist/locations.py

```diff
@@ -0,0 +1,18 @@
+import sys
+
+if sys.version_info >= (3, 12):
+    from distutils.sysconfig import *
+    __all__ = ['get_python_inc', 'get_config_var', "get_python_version", "PREFIX", "get_python_lib", "get_config_vars"]
+else:
+    from sysconfig import *
+
+    __all__ = ['get_python_inc', 'get_config_var', "get_python_version", "PREFIX", "get_python_lib", "get_config_vars"]
```

Comment:
This is the same in both cases, put this at the top

Also, use consistent quotation marks